tests/all/main.rs

```diff
@@ -122,6 +123,18 @@ pub(crate) fn gc_store() -> Result<wasmtime::Store<()>> {
     Ok(wasmtime::Store::new(&engine, ()))
 }
 
+pub(crate) fn gc_and_exceptions_store() -> Result<wasmtime::Store<()>> {
+    let _ = env_logger::try_init();
+
+    let mut config = wasmtime::Config::new();
+    config.wasm_function_references(true);
+    config.wasm_gc(true);
+    config.wasm_exceptions(true);
+
+    let engine = wasmtime::Engine::new(&config)?;
+    Ok(wasmtime::Store::new(&engine, ()))
+}
```

Comment:
Could this be updated to disable `wasm_gc` and `wasm_function_references`, or not explictily enable them, since they shouldn't be required?

Reply:
Done!